src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    { person.age !== undefined && (
+    <p className="Person__age">
+      {`I am ${person.age}`}
+    </p>
+    )}
+
+    <p className="Person__partner">
+      {!person.isMarried
+        ? `I am not married`
+        : `${person.partnerName} is my ${getTitle(person)}`}
+    </p>
+  </section>
+);
+
+const getTitle = person => (person.sex === 'f' ? `husband` : `wife`);
```

Comment:
if you want to use this method in a component, you need to declare this method inside this component